src/gov/nasa/worldwind/symbology/milstd2525/graphics/areas/SearchArea.java

```diff
@@ -496,9 +496,8 @@ protected void determineActiveAttributes()
     protected Path createPath(List<Position> positions)
     {
         Path path = new Path(positions);
-        path.setFollowTerrain(true);
+        path.setSurfacePath(true);
         path.setPathType(AVKey.GREAT_CIRCLE);
-        path.setAltitudeMode(WorldWind.CLAMP_TO_GROUND);
```

Comment:
the `setAltitudeMode` function is also used in our other platforms (for instance, in Web WorldWind). Are we sure we want to get rid of this?

Reply:
The call to setSurfacePath on line 499 turns on clamp to ground. setAltitudeMode is still available on the Path class. That being said, please create an issue in WWW to make sure the designs of WWW and WWJ are aligned with respect to Paths and Symbology.

Reply:
Will do

Reply:
[Issue #817 created in WebWW repo.](https://github.com/NASAWorldWind/WebWorldWind/issues/817)